articles/cosmos-db/unique-keys.md

```diff
@@ -130,6 +130,20 @@ The following command sample shows how to create a unique index on the firstName
 ```
 db.users.createIndex( { firstName: 1, lastName: 1, email: 1 }, { unique: true } )
 ```
+## Azure portal sample
```

Comment:
@jantielens Can you please update this to:

## Configure unique keys by using Azure portal

Reply:
Also can you please update the image to this: 
![image](https://user-images.githubusercontent.com/14284405/40400677-1815a9ca-5df8-11e8-9c5e-c07924a1d595.png)